lib/commands/general.js

```diff
@@ -202,24 +201,14 @@ const commands = {
    * @this {XCUITestDriver}
    */
   async launchApp() {
-    const APP_LAUNCH_TIMEOUT = 20 * 1000;
-
-    this.logEvent('appLaunchAttempted');
-    await this.opts.device.launchApp(this.opts.bundleId);
-
-    let checkStatus = async () => {
-      let response = await this.proxyCommand('/status', 'GET');
-      let currentApp = response.currentApp.bundleID;
-      if (currentApp !== this.opts.bundleId) {
-        throw new Error(`${this.opts.bundleId} not in foreground. ${currentApp} is in foreground`);
-      }
-    };
-
-    this.log.info(`Waiting for '${this.opts.bundleId}' to be in foreground`);
-    let retries = parseInt(APP_LAUNCH_TIMEOUT / 200, 10);
-    await retryInterval(retries, 200, checkStatus);
-    this.log.info(`${this.opts.bundleId} is in foreground`);
-    this.logEvent('appLaunched');
+    const appName = this.opts.app || this.opts.bundleId;
+    try {
+      await this.start();
```

Comment:
I'm not sure this is what we need to do. The `start` method also does some manipulations to WDA and web views.